src/ai/backend/manager/models/alembic/versions/1d42c726d8a3_migrate_container_registries_from_etcd_to_psql.py

```diff
@@ -346,6 +454,7 @@ def upgrade():
     )
 
     insert_registry_id_to_images()
+    insert_registry_id_to_images_with_black_registry_id()
```

Comment:
Seems to be like a typo: `black` → `blank`?
More precisely, it is not "blank" because there is original value corresponding to the project part in the image ref but just there is no corresponding registry information. "missing" would fit better here.